lib/fluent/plugin/in_sample.rb

```diff
@@ -39,6 +39,8 @@ class SampleInput < Input
     config_param :auto_increment_key, :string, default: nil
     desc "The boolean to suspend-and-resume incremental value after restart"
     config_param :suspend, :bool, default: false,deprecated: 'This parameters is ignored'
+    desc "The boolean to control how to generate a record. You can enable it if filter does not do destructive change"
```

Comment:
It would be easier to understand if this explained what this option does and its purpose.

```suggestion
    desc "Reuse the sample data to reduce the load when sending large amounts of data. You can enable it if filter does not do destructive change"
```